File: routeMap/app/src/main/java/ru/android/zheka/vm/IPanelHomeVM.java
package ru.android.zheka.vm;

import ru.android.zheka.coreUI.IVM;
import ru.android.zheka.model.IHomeModel;

public interface IPanelHomeVM extends IVM <IHomeModel> {
    void info();

    void address();

    void geo();

    void editPoints();

    void editTraces();

    void pointNavigate();

    void createTrace();

    void settingsAction();
}

File: routeMap/app/src/main/java/ru/android/zheka/vm/IPanelHomeVM.kt
package ru.android.zheka.vm

import ru.android.zheka.coreUI.IVM
import ru.android.zheka.model.IHomeModel

interface IPanelHomeVM : IVM<IHomeModel?> {
    fun info()
    fun address()
    fun geo()
    fun editPoints()
    fun editTraces()
    fun pointNavigate()
    fun createTrace()
}